bash_completionsV2.go

```diff
@@ -224,20 +225,28 @@ __%[1]s_handle_completion_types() {
         # completions at once on the command-line we must remove the descriptions.
         # https://github.com/spf13/cobra/issues/1508
         local tab=$'\t' comp
-        while IFS='' read -r comp; do
+        for comp in "${completions[@]}"; do
             [[ -z $comp ]] && continue
             # Strip any description
             comp=${comp%%%%$tab*}
             # Only consider the completions that match
             if [[ $comp == "$cur"* ]]; then
                 COMPREPLY+=("$comp")
-            fi
-        done < <(printf "%%s\n" "${completions[@]}")
+           fi
+        done
+
+        __%[1]s_escape_compreply
+        ;;
+
+    63)
+        # Type: Listing completions after successive tabs
+        __%[1]s_handle_standard_completion_case
         ;;
 
     *)
         # Type: complete (normal completion)
         __%[1]s_handle_standard_completion_case
+        __%[1]s_escape_compreply
```

Comment:
I liked the elegance of trying to escape the characters at the end.  But it turned out we also needed to escape them from within `__%[1]s_handle_standard_completion_case`, so I decided to do everything from within that function.